tools/docker-compose/ansible_nightly.repo

```diff
@@ -0,0 +1,4 @@
+[ansible-nightly]
+baseurl=https://releases.ansible.com/ansible/rpm/nightly/devel/epel-7-$basearch
```

Comment:
So this makes the container's default version devel??? I remember @kdelee talked about doing this with the inventory plugins, but that would involve changing the production container image. Seeing as how this keeps coming back up, I'd vote to always make AWX Ansible devel by default....